.dockerignore

```diff
@@ -0,0 +1,11 @@
+.git
```

Comment:
This is to help with image caching, otherwise any file changes will cause a cache miss.

Reply:
Funny, I had through docker automatically ignores that. Go figure.